vt/client.py

```diff
@@ -233,7 +235,8 @@ async def _response_to_json(self, response):
     error = await self.get_error_async(response)
     if error:
       raise error
-    return await response.json_async()
+    response_text = await response.read_async()
+    return json.loads(response_text)
```

Comment:
This does not make any sense to me.  Why did you replace `return await response.json_async()` here?  You replaced it with `response.read_async()` which will return a truncated content, my current modification logs the result of calling this and it is also truncated, so `json.load(response_text)` will fail due to `response_text` being truncated.

I recommend that you keep the original code on line 236, since you modified the `json_async()` method...

Reply:
Agreed, you're right :)